README.md

```diff
@@ -62,6 +62,7 @@ Set up your playground and check out the getting started examples, which will gu
 * [Offloading a Service](https://docs.liqo.io/en/stable/examples/service-offloading.html): learn how to create a multi-cluster service, and how to consume it from each connected cluster.
 * [Stateful Applications](https://docs.liqo.io/en/stable/examples/stateful-applications.html): find out how to deploy a database across a multi-cluster environment, leveraging the Liqo storage fabric.
 * [Global Ingress](https://docs.liqo.io/en/stable/examples/global-ingress.html): discover how route external traffic to multi-cluster applications through a global ingress and automatic DNS configurations.
+* [Replicated Deployments](https://docs.liqo.io/en/stable/examples/replicated-deployments.html): learn how to offload an application on multiple remote clusters, replicating its pods on all the available clusters.
```

Comment:
Can you add an entry for the terraform example, please? Sorry, we missed it with the right pr